src/index.js

```diff
@@ -171,14 +171,20 @@ function iterateBody(self, b, dt, noGravity) {
 
     // linear air or fluid friction - effectively v *= drag
     // body settings override global settings
-    var drag = (b.airDrag >= 0) ? b.airDrag : self.airDrag
-    if (b.inFluid) {
-        drag = (b.fluidDrag >= 0) ? b.fluidDrag : self.fluidDrag
-        drag *= 1 - (1 - b.ratioInFluid) ** 2
+    var getDrag = (axisFluidDrag) => {
```

Comment:
dont create a func in here as that'll create unwanted garbage, make the func outside and pass in needed args